.mailmap

```diff
@@ -78,7 +78,7 @@ Gaurav Gautam <gautam1168@gmail.com> gautam1168 <gautam1168@gmail.com>
 Gerrit Leck <gerritleck@web.de>
 Gerrit Leck <gerritleck@web.de> Gerrit Leck <g.leck@gsi.de>
 
-Isaac Smith <smithis7@msu.edu> 
+Isaac Smith <smithis7@msu.edu>
```

Comment:
Why does this line change? Should be cleaned up.

Reply:
there was a space in the end and fixed by the pre-commit. 200 files have similar problems, if use `pre-commit run --all-files`, but not necessary to scan all now.